### ISSUE TITLE
[1.x] Hide DEPRECATION WARNINGs in 3rd-party libraries (node_modules)

### ISSUE THREAD

**Issue by caugner** [sass/dart-sass#1349]
There is an imminent need to reduce the noise and build time cost of the `DEPRECATION WARNING`s introduced in Sass 1.33. My frontend builds still take more than twice as long with Sass 1.34, which already reduced duplicate warnings. This is not only annoying (and confusing for junior devs), but also causing unnecessary CO2 emissions.

If you ask me, **Sass should distinguish warnings** in own `*.scss` files versus warnings in 3rd-party libraries (`node_modules/*.scss`), and ignore these at all or only omit the first warning for each 3rd-party library. Otherwise you're essentially addressing the wrong audience. Folks like me who use Bootstrap v4 **cannot** fix those warnings ourselves, and upgrading to Bootstrap v5 is not a viable option for many of us. In other words, those warnings belong in the Bootstrap v4 builds, but not in the builds of Bootstrap v4 users.

Anticipating breaking changes is awesome, but well meant is not necessarily well done. These warnings cause harm in their current form (even after the mitigations in 1.34). And, last but not least, let's hope that the 1.x line doesn't require any security fix, because those folks that are downgrading to 1.32 in the masses to workaround the warnings will miss that.

tl;dr **Please rework the deprecation warnings** to reduce the impact for end-users, I'm begging you.

**Comment by nex3** [comment]
Duplicate of https://github.com/sass/dart-sass/issues/672. **tl;dr:** There's already a `--quiet-deps` flag on the CLI and in the Dart API, we're working on making this available to the JS API as well.

**Comment by OzzyTheGiant** [comment]
Was wondering why this was happening across all my projects. For anyone still interested in fixing the deprecation warnings temporarily, without wasting too much time messing with configurations, try locking your sass dependency to 1.30.x as that's the last version I was able to use without the warnings popping up everywhere. So your package.json should look like:

```
{
    devDependencies: {
        "sass": "~1.30.0"
        ...
    }
}
```

**Comment by nex3** [comment]
We strongly recommend against locking Sass to an older version. CSS is constantly changing, and you *will* run into issues where you try to use new CSS features and they simply fail.
